src/hugr/rewrite/outline_cfg.rs

```diff
@@ -140,13 +143,16 @@ impl Rewrite for OutlineCfg {
         };
 
         // 3. Extract Cfg node created above (it moved when we called insert_hugr)
-        let cfg_node = h
+        // Support filtered Sibling-only views by explicitly descending into new_block
```

Comment:
This is to show the kind of thing I'm thinking of. We can't do everything this way - eventually we have to just break out to the `hugr_mut(): &mut Hugr` - but we can do *most* things remaining within one (nested) SiblingGraph at a time

Reply:
However, worth noting that an alternative for these bits is just for `insert_hugr` to return the NodeIndex->NodeIndex map from `insert_hugr_internal`!....should I do that instead?

Reply:
Yes, this will get significantly easier after #535. Maybe wait for that to go in first....